### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#501] [Esami & Esoneri M-Z]
### Traccia

<img width="610" alt="image" src="https://github.com/user-attachments/assets/8c70ccb8-c757-4d01-8d31-0c47942e3b32">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/bc48abc0-53dd-452f-9a71-b17079ace309)
